src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,63 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private ColorSupplier colorSupplier;
+    private Random random = new Random();
+    private final int maxSide = 10;
+    private final int minSide = 1;
+    private final String defaultColor = "WHITE";
```

Comment:
These have to be constants. See the article to find out what is constant and what requirements for naming
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names
![image](https://github.com/mate-academy/jv-oop-advanced/assets/20326858/e9e7d0c9-9015-4af2-afcb-c5eaabeb40a6)